cookbooks/add_memory_using_qdrant_cloud.py

```diff
@@ -0,0 +1,40 @@
+# This example shows how to use vector config to use QDRANT CLOUD
+import os
+from dotenv import load_dotenv
+from mem0 import Memory
+
+# Loading my API_KEY for OPENAI
+load_dotenv()
+OPENAI_API_KEY = os.environ.get('OPENAI_API_KEY')
+USER_ID = "rajib"
+os.environ['OPENAI_API_KEY'] = OPENAI_API_KEY
+
+# creating the config attributes
+collection_name="memory" # this is the collection I created in QDRABT cloud
+api_key=os.environ.get("QDRANT_API_KEY") # Getting the QDRANT api KEY
+host="668a5b8f-6635-45c1-b30f-41c6c2164929.us-east4-0.gcp.cloud.qdrant.io" #QDRANT HOST
```

Comment:
Also here you do replace URL with `qdrant_host`

Reply:
@rajib76 Here I'm suggesting that you can remove this `668a5b8f-6635-45c1-b30f-41c6c2164929.us-east4-0.gcp.cloud.qdrant.io` and write something like `xx.gcp.cloud.qdrant.io`

Reply:
done